src/Card/Card.scss

```diff
@@ -5,7 +5,31 @@
 .pgn__card-body {
   width: 100%;
 
-  &.is-muted {
+  &.pgn__card-dark {
+    border: 0;
+    background-color: $primary-500;
+    color: $white;
+
+    .pgn__card-header {
+      .pgn__card-header-title,
+      .pgn__card-header-title-sm,
+      .pgn__card-header-title-md {
+        color: $white;
```

Comment:
@viktorrusakov @monteri We will need to handle if/when a `title` and/or `subtitle` have a nested hyperlink in the `variant="dark"` use case, e.g.:

```jsx
<Card variant="dark">
  <Card.Header
    title={(
      <Hyperlink
        destination="https://google.com"
        target="_blank"
      >
        Card Title
       </Hyperlink>
    )}
  />
</Card>
```

<img width="338" alt="image" src="https://user-images.githubusercontent.com/2828721/208545028-14f6ab03-da09-4986-ba41-bef8679fb204.png">